webextension/background.js

```diff
@@ -141,10 +146,15 @@ const TabState = (function() {
       this._reportSubmitPromise = new Promise(async (resolve, reject) => {
         const report = this._report;
         const { incognito, url } = await browser.tabs.get(this._tabId);
-        if (incognito) {
+        if (incognito !== undefined) {
```

Comment:
So, this raises an interesting point... we're never going to prompt users if they're incognito. BUT, should we hide the icon for private windows, or let them opt-into reports?

@rosanardila, thoughts?